src/development/development.controller.ts

```diff
@@ -5,30 +5,24 @@ import { Response } from "express";
 import { ReSeedSuccessResponse } from "./development.response";
 import {
     ServerErrorResponse,
-    UnauthorizedErrorResponse,
     UnprocessableEntityErrorResponse,
 } from "../global/responses/errors";
+import { Public } from "@/global/decorators/public.decorator";
 
 @Controller("development")
 @ApiTags("Development")
 export class DevelopmentController {
     constructor(private readonly developmentService: DevelopmentService) {}
 
     @ApiOperation({
-        summary: "Reseed the database",
-        description:
-            "It will take a while, maybe minutes. Then you'll be logged out.",
+        summary: "Public Route: Reseed the database",
+        description: "It will take a while, maybe minutes",
```

Comment:
can we also keep this so people know they will have to log in again
"Then you'll be logged out."